app/css/interceptor.css

```
@@ -48,7 +48,7 @@ button:where(:not(.btn)):hover {
 	background-color: var(--highlighted-primary-color);
 }
 
-button:not(.btn):disabled, button[disabled]) {
+button:not(.btn):disabled, button[disabled] {
```

Should also prevent matching `.btn` classes on the second selector

```suggestion
button:not(.btn):disabled, button:not(.btn)[disabled] {
```